docs/reference/remap/functions/add_datadog_tags.cue

```diff
@@ -0,0 +1,45 @@
+package metadata
+
+remap: functions: add_datadog_tags: {
```

Comment:
I wonder if we should call this `datadog_add_tags`, to prepare for more functions that are specific to datadog, or other products (e.g. `<product>_...`).

We do have a couple of AWS specific functions, but those are all parse functions, so they start with `parse_aws_...`.